packages/contracts-bedrock/scripts/libraries/ForgeArtifacts.sol

```diff
@@ -128,6 +190,14 @@ library ForgeArtifacts {
     function getInitializedSlot(string memory _contractName) internal returns (StorageSlot memory slot_) {
         string memory storageLayout = getStorageLayout(_contractName);
 
+        // FaultDisputeGame and PermissionedDisputeGame use a different name for the initialized storage slot.
+        string memory slotName = "_initialized";
+        string memory slotType = "t_uint8";
+        if (LibString.eq(_contractName, "FaultDisputeGame") || LibString.eq(_contractName, "PermissionedDisputeGame")) {
```

Comment:
This isn't scalable. Maybe we do an or in the `slotName` and `slotType` ?

Reply:
I guess so, I personally think this is actually a sign that this variable name/type should be changed so that it matches the other variables, keeping this here is a reminder of the difference instead of hiding it.

Reply:
Is the difference oversight, or because of different initializer libraries? I know we're gradually migrating to OZ v5 in which case it wouldn't surprise me to see the different names and type.

As long as we're separately tracking unifying all initializer approaches I'm ok with both using an "or" and  keeping this explicit list as a reminder

Reply:
Nah it's a custom initializer entirely

Reply:
It probably should just be OZ